app/build.gradle

```diff
@@ -220,7 +220,7 @@ dependencies {
 
     // MVP library
     compile "com.hannesdorfmann.mosby3:mvp:$versions.mosbyMvpVersion"
-    compile "com.hannesdorfmann.mosby3:viewstate:$versions.mosbyMvpVersion"
+    compile "com.hannesdorfmann.mosby3:viewstate:3.0.1" //freaky bug won't work with a non-hardcoded version
```

Comment:
I tried with your piece of code and it works now. Thanks! I had the exact same thing, something reallly weird must have been happening